packages/context/src/context-view.ts

```diff
@@ -28,23 +29,34 @@ const nextTick = promisify(process.nextTick);
  * points. For example, the RestServer can react to `controller` bindings even
  * they are added/removed/updated after the application starts.
  *
+ * `ContextView` is an event emitter that emits the following events:
+ * - 'close': when the view is closed (stopped observing context events)
+ * - 'refresh': when the view is refreshed as bindings are added/removed
+ * - 'resolve': when the cached values are resolved and updated
  */
-export class ContextView<T = unknown> implements ContextObserver {
+export class ContextView<T = unknown> extends EventEmitter
+  implements ContextObserver {
   protected _cachedBindings: Readonly<Binding<T>>[] | undefined;
   protected _cachedValues: T[] | undefined;
   private _subscription: Subscription | undefined;
 
   constructor(
     protected readonly context: Context,
     public readonly filter: BindingFilter,
-  ) {}
+  ) {
+    super();
```

Comment:
Is it necessary to explicitly call the parent constructor? Isn't it automatically called by JS runtime?

Reply:
Yes, it's necessary as the base class's constructor has a different signature.